src/nvim/window.c

```diff
@@ -853,11 +853,11 @@ void ui_ext_win_position(win_T *wp)
       bool east = c.anchor & kFloatAnchorEast;
       bool south = c.anchor & kFloatAnchorSouth;
 
-      int comp_row = (int)row - (south ? wp->w_height : 0);
-      int comp_col = (int)col - (east ? wp->w_width : 0);
+      int comp_row = (int)row - (south ? wp->w_height_outer - 1 : 0);
```

Comment:
why -1 here? It doesn't seem right.

Reply:
The difference in coordinates between top and botton sides is `w_height_outer - 1`.

Reply:
I'm pretty sure that difference is `w_height_outer`. That's what the variable is for.

Reply:
With the `- 1`:
![Screenshot_20211001_154139](https://user-images.githubusercontent.com/35768171/135584725-d47fe263-eadb-4685-8747-70a754efbb9e.png)
![Screenshot_20211001_154157](https://user-images.githubusercontent.com/35768171/135584731-949a57f1-344f-43ff-a7d9-9c131d9c3b03.png)
![Screenshot_20211001_154215](https://user-images.githubusercontent.com/35768171/135584740-ee418745-7f98-424e-8c13-744038e6dd47.png)
![Screenshot_20211001_154230](https://user-images.githubusercontent.com/35768171/135584746-a43a3190-bc2e-44aa-bf96-f3222862fecf.png)

Without the `- 1`:
![Screenshot_20211001_154654](https://user-images.githubusercontent.com/35768171/135584768-621931b8-cf41-456a-a77f-00c9a739251a.png)
![Screenshot_20211001_154708](https://user-images.githubusercontent.com/35768171/135584785-ce47b173-b581-40ba-b6d8-d742f54e80dd.png)
![Screenshot_20211001_154722](https://user-images.githubusercontent.com/35768171/135584818-613154db-33ec-4166-9763-0274f5672152.png)
![Screenshot_20211001_154736](https://user-images.githubusercontent.com/35768171/135584833-1ced1b76-016e-4677-b916-e4b4d071bf2a.png)

Which one is intended?

Reply:
the latter. think of the anchor point as the upper-left corner of the marked cell, not the middle of it.

Reply:
OK, I change it.